### DISCUSSION TITLE
How to use `react-tooltip` with `react-simple-maps`?

### DISCUSSION THREAD

**Discussion by bnbon** [ReactTooltip/react-tooltip#910] [Q&A] [answered]
I wonder if anyone has used this with `react-simple-maps` yet, that is how I heard of this package, but their documentation covers v4.

I tried doing the conversion of the import and changing the <ReactTooltip>Content<ReactTooltip> to <Tooltip content="Content"/> but I also think I need to add a pair of ID's for every country? I'd rather use the single instance of a tooltip and set the label to the country dynamically – do I need to use the Provider approach?

Many Thanks

**Comment by danielbarion** [comment]
Hi @bonbonio, please take a look at this section in our docs: https://react-tooltip.com/docs/examples/multiple-anchors

**Comment by eannam** [comment]
Has anyone managed to get this working?

**Comment by gabrieljablonski** [comment, marked as answer]
@eannam Someone posted a [working example](https://github.com/zcreativelabs/react-simple-maps/issues/314#issuecomment-1406462994) over at the `react-simple-maps` repo.
